frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -101,7 +89,11 @@ export class RhvImportWizard extends ImportWizard {
     // Clicking `edit` button to reach network and storage settings
     await click(rhvView.editButton);
     await click(view.nextButton);
-    await this.next();
+    await click(view.nextButton);
```

Comment:
this seems very prone to breaking. Why are we doing this? Can we wait on some condition. Is `Please wait for a VM to load.` error getting triggered with the first next?

Reply:
No that error, cause I'm waiting until spinner will end. If I leave only one click on `next` nothing will happen, button will not be pressed. I have no explanation why.

Reply:
so it seems this is working purely by coincidence or timing on your env. Please debug it and introduce some consistent solution.

Reply:
Fixed by adding timeout to make sure this button will be clickable